testsuite/config/cucumber.yml

```diff
@@ -37,3 +37,4 @@ xmlrpc: --tags @scope_xmlrpc
 power_management: --tags @scope_power_management
 retracted_patches: --tags @scope_retracted_patches
 ansible: --tags @scope_ansible
+reportdb: --tags @scope_reportdb
```

Comment:
Be aware that you also need to add an entry in all the CI pipelines.
See example:
https://github.com/SUSE/susemanager-ci/blob/master/jenkins_pipelines/environments/manager-4.2-dev-acceptance-tests-NUE#L24

Reply:
For future reference, done in https://github.com/SUSE/susemanager-ci/pull/513